tests/test_pandas_dataset.py

```diff
@@ -957,6 +957,21 @@ def expectation_that_crashes_on_sixes(self, column):
         self.assertTrue(type(df2) == type(df))
 
 
+    def test_validate_map_expectation_on_categorical_column(self):
```

Comment:
Added a very simple test to verify that an exception that was previously thrown on validating a categorical expectation is not happening anymore.